it/mail/chrome/messenger/FilterEditor.dtd

```diff
@@ -39,49 +39,49 @@
 <!ENTITY copyMessage.label "Copia il messaggio in">
 <!ENTITY forwardTo.label "Inoltra il messaggio a">
 <!ENTITY replyWithTemplate.label "Rispondi con un modello">
-<!ENTITY markMessageRead.label "Marca il messaggio come letto">
-<!ENTITY markMessageStarred.label "Aggiungi a Speciali">
-<!ENTITY labelMessage.label "Label Message As">
+<!ENTITY markMessageRead.label "Contrassegna come letto">
+<!ENTITY markMessageStarred.label "Aggiungi stelle">
+<!ENTITY labelMessage.label "Etichetta il messaggio come">
 <!ENTITY setPriority.label "Imposta priorità a">
 <!ENTITY addTag.label "Etichetta il messaggio">
-<!ENTITY setJunkScore.label "Considera il messaggio">
+<!ENTITY setJunkScore.label "Imposta stato Indesiderata come">
 <!ENTITY deleteMessage.label "Elimina messaggio">
 <!ENTITY deleteFromPOP.label "Elimina dal server POP">
-<!ENTITY fetchFromPOP.label "Scarica dal server POP">
+<!ENTITY fetchFromPOP.label "Preleva dal server POP">
 <!ENTITY ignoreThread.label "Ignora la discussione">
-<!ENTITY watchThread.label "Tieni d'occhio la discussione">
+<!ENTITY watchThread.label "Guaarda la discussione">
 <!ENTITY stopExecution.label "Interrompi l'esecuzione dei filtri">
 
 
 <!--
  entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY contextDesc.label "Apply filter when:">
-<!ENTITY contextManual.label "Manually Run">
-<!ENTITY contextBoth.label "Checking Mail or Manually Run">
-<!ENTITY contextIncoming.label "Checking Mail">
+<!ENTITY contextDesc.label "Applica filtro quando:">
+<!ENTITY contextManual.label "Esecuzione manuale">
+<!ENTITY contextBoth.label "Controllo e-mail or Esecuzione manuale">
+<!ENTITY contextIncoming.label "Controllo e-mail">
 <!ENTITY contextDesc.accesskey "w">
 
 
 <!--
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY playSound.label "Play Sound">
-<!ENTITY notPendingStatus.label "Not Pending">
-<!ENTITY noSoundSelected.label "Select a Sound">
-<!ENTITY noScriptsInstalled.label "No Scripts installed">
+<!ENTITY playSound.label "Riproduci suono">
+<!ENTITY notPendingStatus.label "Non in attesa">
+<!ENTITY noSoundSelected.label "Seleziona un suono">
+<!ENTITY noScriptsInstalled.label "Nessun script installato">
```

Comment:
CORRECTIONS NEEDED. 

There is a typo. Please replace "Nessun" with "Nessuno".